app/controllers/wpcc/engine_controller.rb

```diff
@@ -3,7 +3,7 @@ class EngineController < Wpcc.parent_controller.constantize
     protect_from_forgery
 
     layout 'wpcc/engine'
-    before_action :log_session
+    before_action :log_session, :check_wpcc_session
```

Comment:
My understanding of the explicit use of wpcc in the name is to ensure that the session is not complicated by other tools that use sessions. See @tomas-stefano comment near the top of this PR.